css/section-2.css

```diff
@@ -0,0 +1,28 @@
+.who {
```

Comment:
Add all these changes in `about.css`

Reply:
okay i will make change then push it again